### DISCUSSION TITLE
How connect with postgre using ssh tunnel?

### DISCUSSION THREAD

**Discussion by mxcdh** [prisma/prisma#12993] [Q&A]
Hello

How I can connect with my database via ssh tunnel?
Link for ssh tunnel look like:
```
postgresql+ssh://sshUser:sshPassword@sshServerUrl/dbUser:dbPassword@dbUrl:dbPort/dbName
```
Can Prisma handle this format?

**Comment by mxcdh** [comment]
Anyone can help me?

**Comment by austincrim** [comment]
Hey @mxcdh 👋 ,

I don't think this is supported in Prisma today. See [this similar issue](https://github.com/prisma/prisma/issues/10019) for MySQL.

Would you mind opening a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) for this?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
